src/experiments/tests/prebid-shared-id.ts

```diff
@@ -0,0 +1,28 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const prebidSharedId: ABTest = {
+	id: 'PrebidSharedId',
+	author: '@commercial-dev',
+	start: '2024-09-25',
+	expiry: '2024-10-30',
+	audience: 0 / 100,
```

Comment:
Are we setting it to 50% in a later PR?

Reply:
whoops I changed `audienceOffset` by mistake! Nice spot thanks